src/main/java/sirius/kernel/commons/Files.java

```diff
@@ -146,6 +159,34 @@ public static String getFilenameWithoutExtension(@Nullable String path) {
         return Strings.splitAtLast(getFilenameAndExtension(path), ".").getFirst();
     }
 
+    /**
+     * Determines if the given path is hidden.
+     * <p>
+     * A path is considered hidden if it starts with a dot.
+     *
+     * @param path the path to check
+     * @return <tt>true</tt> if the path is hidden, <tt>false</tt> otherwise
+     */
+    public static boolean isHidden(@Nullable String path) {
+        return path != null && path.startsWith(".");
```

Comment:
**Required:** One more thing, we are talking about a path here. Eg: `folder1/folder2/folder3/.hiddenfile`
We need to check if the final portion of the path is considered a hidden file. You can use Files.getFilenameAndExtension to obtain the last portion of it, and there check if it starts with `.`